doc/snippets/Microsoft.Data.SqlClient/SqlClientPermission.xml

```diff
@@ -1,153 +1,166 @@
+        </note>
+      </remarks>
+    </SqlClientPermission>
+    <ctor name="default">
+      <summary>
+        Initializes a new instance of the <see cref="T:Microsoft.Data.SqlClient.SqlClientPermission" /> class.
+      </summary>
+    </ctor>
+    <ctor name="PermissionState">
+      <param name="state">
+        One of the <see cref="T:System.Security.Permissions.PermissionState" /> values.
+      </param>
+      <summary>
+        Initializes a new instance of the <see cref="T:Microsoft.Data.SqlClient.SqlClientPermission" /> class.
+      </summary>
+    </ctor>
+    <ctor name="PermissionStateAndAllowBlankPasswordBool">
+      <param name="state">
+        One of the <see cref="T:System.Security.Permissions.PermissionState" /> values.
+      </param>
+      <param name="allowBlankPassword">
+        Indicates whether a blank password is allowed.
+      </param>
+      <summary>
+        Initializes a new instance of the <see cref="T:Microsoft.Data.SqlClient.SqlClientPermission" /> class.
+      </summary>
+      <remarks>
+          The <see cref="T:System.Security.Permissions.PermissionState" /> enumeration takes precedence over the <see cref="P:System.Data.Common.DBDataPermission.AllowBlankPassword" /> property. Therefore, if you set <see cref="P:System.Data.Common.DBDataPermission.AllowBlankPassword" /> to <see langword="false" />, you must also set <see cref="T:System.Security.Permissions.PermissionState" /> to <c>None</c> to prevent a user from making a connection using a blank password. For an example demonstrating how to use security demands, see <see href='/dotnet/framework/data/adonet/code-access-security'>Code Access Security and ADO.NET</see>.
+      </remarks>
+    </ctor>
+    <Add name="connectionStringAndRestrictionsStringAndBehavior">
+      <param name="connectionString">
+        The connection string.
+      </param>
+      <param name="restrictions">
+        The key restrictions.
+      </param>
+      <param name="behavior">
+        One of the <see cref="T:System.Data.KeyRestrictionBehavior" /> enumerations.
+      </param>
+      <summary>
+        Adds a new connection string and a set of restricted keywords to the <see cref="T:Microsoft.Data.SqlClient.SqlClientPermission" /> object.
+      </summary>
+      <remarks>
+        <para>
+          Use this method to configure which connection strings are allowed by a particular permission object. For example, use the following code fragment if you want to only allow a specific connection string and nothing else:
+        </para>
+        <para>
+          <c>permission.Add("server=MyServer; database=MyDatabase; Integrated Security=true", "", KeyRestrictionBehavior.AllowOnly)</c>
```

Comment:
Isn't this code in c#?

Reply:
Yeah, that's a good point. Since it's a paragraph with just inline code in it, it can be replaced with a code block.